plugins/web-plugin/src/main/java/ru/sbtqa/tag/stepdefs/en/WebSteps.java

```diff
@@ -84,4 +86,53 @@ public void goToPageByUrl(String url) throws PageInitializationException {
     public void reInitPage() {
         super.reInitPage();
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @And("^user accepts alert with text \"(.*?)\"$")
+    public void acceptAlert(String text) throws WaitException {
+        super.acceptAlert(text);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @And("^user dismisses alert with text \"(.*?)\"$")
+    public void dismissAlert(String text) throws WaitException {
+        super.dismissAlert(text);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @And("^user checks that text \"(.*?)\" appears on the page$")
+    public void checkTextAppears(String text) throws WaitException {
+        super.checkTextAppears(text);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @And("^user checks that text \"(.*?)\" is absent on the page$")
+    public void checkTextIsNotPresent(String text) {
+        super.checkTextIsNotPresent(text);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @And("^user checks that modal window with text \"(.*?)\" is appears$")
+    public void checkModalWindowAppears(String text) throws WaitException {
+        super.checkModalWindowAppears(text);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @And("^user checks that element with text \"(.*?)\" is present$")
+    @When("^user checks that the text \"(.*?)\" is visible$")
+    public void checkElementWithTextIsPresent(String text) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add the "@Override" annotation above this method signature ([squid:S1161](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1161))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=96&issues=AWZh47DWrFHbA8-Egpm6&open=AWZh47DWrFHbA8-Egpm6)

[ISSUE_KEY]: AWZh47DWrFHbA8-Egpm6